public/index.html

```diff
@@ -217,16 +218,20 @@
           <section class='col-xs-12 col-sm-6 col-md-6'>
             <section>
               <h2>Deploying code changes</h2>
```

Comment:
@bparees if you fork the repo, clone and use the templates to deploy to OpenShift, you will actually deploy the code from openshift/rail-ex, and not your fork. Without changing the SOURCE_REPOSITORY_URL parameter, your webhooks will trigger rebuilds of openshift/rail-ex no matter what code changes you've made and pushed into your fork.

I think we we to address this by explaining this required step. What do you say? Where would it go?